plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version3/class-wc-rest-taxes-controller-tests.php

```diff
@@ -277,12 +277,39 @@ public function test_can_get_taxes_filtering_by_class( $class ) {
 		$this->assertEquals( 200, $response->get_status() );
 		$data = array_values( $response->get_data() );
 		$ids  = array_map(
-			function( $item ) {
+			function ( $item ) {
 				return $item['id'];
 			},
 			$data
 		);
 
 		$this->assertEquals( array( $tax_ids_by_class[ $class ] ), $ids );
 	}
+
+	/**
+	 * @testdox Tax rates with non-Latin characters in tax class names are properly created and associated with the correct class.
+	 */
+	public function test_can_create_tax_rate_with_non_latin_tax_class() {
+		wp_set_current_user( $this->user );
+
+		$tax_class_name = '∑';
```

Comment:
Question: do we need to check and add test for other none-Latin class names? such as Arabic, Persian, Hebrew?

Reply:
Not needed, any non-ASCII/non-Latin produces the same URL-encoded output.